email-service/src/main/java/org/java/mentorship/email/EmailApplication.java

```diff
@@ -7,5 +7,7 @@
 @SpringBootApplication
 public class EmailApplication {
 
-    public static void main(String[] args) { ApplicationContext ctx = SpringApplication.run(EmailApplication.class, args); }
+    public static void main(String[] args) {
+        ApplicationContext ctx = SpringApplication.run(EmailApplication.class, args);
```

Comment:
Since you modified this line :)

We don't really need the variable here
I think you can just do "SpringApplication.run(EmailApplication.class, args);"